### ISSUE TITLE
Message repeating when the Bot responds through displayMessageComposer()

### ISSUE THREAD

**Issue by renehw** [deepak786/intercom_flutter#369]
First of all, thank you for the package, it is very useful and easy to use.

When I configure a button with a fixed message, for example: 

"Olá, tenho interesse em obter o plano Consulta para utilizar o aplicativo de emissão. Pode me passar mais informações?" - in English: "Hello, I'm interested in obtaining the Consultation plan to use the issuing application. Can you give me more information?"

by displayMessageComposer, when the bot responds to me, the text I sent is displayed again in the typing box, which should be clean because I already sent the message before.

`Package Version: v8.0.1`

flutter doctor -v

```
[✓] Flutter (Channel stable, 3.13.9, on Ubuntu 22.04.3 LTS 6.2.0-35-generic, locale pt_BR.UTF-8)
    • Flutter version 3.13.9 on channel stable at /home/rene/snap/flutter/common/flutter
    • Upstream repository https://github.com/flutter/flutter.git
    • Framework revision d211f42860 (12 days ago), 2023-10-25 13:42:25 -0700
    • Engine revision 0545f8705d
    • Dart version 3.1.5
    • DevTools version 2.25.0

[✓] Android toolchain - develop for Android devices (Android SDK version 34.0.0)
    • Android SDK at /home/rene/Android/Sdk
    • Platform android-34, build-tools 34.0.0
    • Java binary at: /snap/android-studio/128/android-studio/jbr/bin/java
    • Java version OpenJDK Runtime Environment (build 17.0.6+0-17.0.6b829.9-10027231)
    • All Android licenses accepted.

[✓] Chrome - develop for the web
    • Chrome at google-chrome

[✓] Linux toolchain - develop for Linux desktop
    • clang version 10.0.0-4ubuntu1
    • cmake version 3.16.3
    • ninja version 1.10.0
    • pkg-config version 0.29.1

[✓] Android Studio (version 2022.3)
    • Android Studio at /snap/android-studio/128/android-studio
    • Flutter plugin can be installed from:
      🔨 https://plugins.jetbrains.com/plugin/9212-flutter
    • Dart plugin can be installed from:
      🔨 https://plugins.jetbrains.com/plugin/6351-dart
    • Java version OpenJDK Runtime Environment (build 17.0.6+0-17.0.6b829.9-10027231)

[✓] VS Code (version unknown)
    • VS Code at /snap/code/current
    • Flutter extension version 3.76.0
    ✗ Unable to determine VS Code version.

[✓] Connected device (3 available)
    • sdk gphone x86 64 (mobile) • emulator-5554 • android-x64    • Android 13 (API 33) (emulator)
    • Linux (desktop)            • linux         • linux-x64      • Ubuntu 22.04.3 LTS 6.2.0-35-generic
    • Chrome (web)               • chrome        • web-javascript • Google Chrome 118.0.5993.117

[✓] Network resources
    • All expected network resources are available.

• No issues found!
```

**Comment by deepak786** [comment]
@renehw It does not matter if the bot has already responded or not. If you open the messenger with a predefined message (using displayMessageComposer), it will always show the provided message. Because you are calling this functionality again and again.

**Comment by renehw** [comment]
But there seems to be an inconsistency... At no point do I close and open the chat, the message appears when the bot responds, apparently when the screen is reloaded, there seems to be a problem with reloading, I haven't looked at the package code, but it seems like When the bot responds, the screen reloads and the message appears to load again.

And as for the method being called several times, I don't think so, because when debugging the app, I put a breakpoint in the method line and it didn't call again.

**Comment by renehw** [comment]
[example.webm](https://github.com/v3rm0n/intercom_flutter/assets/58603535/198d831b-8df1-4920-a105-c4eebaa43222)

**Comment by deepak786** [comment]
This seems to be a bug in the SDK. I will check with Intercom support.

Do you have the same issue on iOS also?

**Comment by renehw** [comment]
I verified here that the problem only occurs on Android, when testing on iOS the problem does not occur.

**Comment by deepak786** [comment]
Can you test with the latest version `intercom_flutter: 8.0.4`?
If you still have the issue then I suggest you create a ticket with intercom support.

**Comment by renehw** [comment]
Didn't solve it with version 8.0.4

**Comment by renehw** [comment]
How do I open a support ticket?

**Comment by deepak786** [comment]
Go to https://www.intercom.com/ and send a message to them from chat icon

**Comment by renehw** [comment]
I contacted the intercom support team, however, the engineers responded and informed me that they cannot provide support as the package is not official. I was already expecting something along these lines, I believe that the correction should be made by the package maintainer, as the problem seems to be just a bug in the rendering of the message in the widget.

Anyway, is there another way to deal with this?

**Comment by deepak786** [comment]
@renehw intercom_flutter package has nothing to do here. It just uses the Intercom native SDKs.
The issue is in the native SDK itself.
I don't know why Intercom support engineer said the above thing. Maybe you shared the flutter code.

Share the code with them in native Android or maybe explain the issue in general without referencing the flutter platform.

**Comment by renehw** [comment]
Thanks for pointing it out, I reported the problem as you said, only dealing with the Android SDK, I'll wait for the answer.

**Comment by renehw** [comment]
Hi @deepak786 !

I spoke directly with the intercom engineering team, I passed on the problem and the video of what happened. The problem has already been found and will be fixed in the next version of the SDK.

**Comment by renehw** [comment]
Hello @deepak786, the intercom team just informed me that the bug was fixed in the latest version of SDK v15.5.0.

Can you update the package to this new version, please?

https://github.com/intercom/intercom-android/releases/tag/15.5.0
